### ISSUE TITLE
RestRequest.responseString Serialization Error on non-UTF8 responses (ISO-8859-1)

### ISSUE THREAD

**Issue by adambergman** [IBM-Swift/SwiftyRequest#14]
It appears that `RestRequest.responseString` has a serialization error because it expects content to be `utf8`.  One example is with google.com which returns `Content-Type: text/html; charset=ISO-8859-1`.   

Example:

```swift
import Foundation
import SwiftyRequest
    let request = RestRequest(method: .get, url: "https://www.google.com")
    request.responseString { response in 
        switch response.result { 
        case .success(let result):
            print("Success \(result)")
        case .failure(let error):
            print("Failure: \(error)")
        }
    }
RunLoop.main.run(until: Date(timeIntervalSinceNow: 15))
```

Output: 

`Failure: Serialization Error`

**Comment by tunniclm** [comment]
@IBM-Swift/kitura This was a problem @adambergman hit and we worked through on Slack. Thanks for opening the issue @adambergman!

**Comment by KyeMaloy97** [comment]
Fixed in PR https://github.com/IBM-Swift/SwiftyRequest/pull/15 and fix available in 1.0.2.
